.github/workflows/esmeta-test262.yml

```
@@ -37,5 +37,4 @@ jobs:
           fi
           echo New or modified test files:
           echo "$paths"
-          "${ESMETA_HOME}"/bin/esmeta test262-test -status -test262dir=$(pwd) $paths
-          
\ No newline at end of file
+          "${ESMETA_HOME}"/bin/esmeta test262-test -status -test262dir=$(pwd) $paths
```

```suggestion
          "${ESMETA_HOME}"/bin/esmeta test262-test -status -test262dir=$(pwd) $paths

```
nit: to be clear, there should always be a trailing newline, but no trailing whitespace :-)